xds/internal/xdsclient/transport/lrs/lrs_stream.go

```diff
@@ -242,7 +242,7 @@ func (lrs *StreamImpl) recvFirstLoadStatsResponse(stream transport.StreamingCall
 	}
 
 	internal := resp.GetLoadReportingInterval()
```

Comment:
I believe the original author meant to name this variable `interval`. Can you please fix the name also?

Reply:
ok